packages/slice-machine/components/DeleteDocumentsDrawer/AssociatedDocumentsCard.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Flex, Text, Card, Link } from "theme-ui";
+
+type CustomtypeDocuments = {
```

Comment:
We'll need to share these types with the backend when we implement it

Reply:
Yep, the type will also likely change since we don't get the CT name from the API